conda/recipes/libcucim/meta.yaml

```diff
@@ -71,7 +71,7 @@ requirements:
     - jbig
     - libwebp-base
     - nvtx-c >=3.1.0
-    - openslide 3
+    - openslide
```

Comment:
No longer needed as `openslide-python` was rebuilt with `4` ( https://github.com/conda-forge/openslide-python-feedstock/pull/17 )

Going forward `openslide` will have migrations, which should hopefully cutdown on the churn here

Reply:
Thanks!